core/runtime.rs

```diff
@@ -184,19 +181,18 @@ pub(crate) struct JsRuntimeState {
   pub(crate) op_ctxs: Box<[OpCtx]>,
   pub(crate) shared_array_buffer_store: Option<SharedArrayBufferStore>,
   pub(crate) compiled_wasm_module_store: Option<CompiledWasmModuleStore>,
-  /// The error that was passed to an explicit `Deno.core.terminate` call.
+  /// The error that was passed to an `op_dispatch_exception` call.
   /// It will be retrieved by `exception_to_err_result` and used as an error
   /// instead of any other exceptions.
-  pub(crate) explicit_terminate_exception: Option<v8::Global<v8::Value>>,
+  // TODO(nayeemrmn): This should be a queue, mirroring `pending_promise_exceptions`.
+  // The REPL doesn't terminate on the first one and should keep track of many.
+  pub(crate) dispatched_exception: Option<v8::Global<v8::Value>>,
+  inspector: Option<Rc<RefCell<JsRuntimeInspector>>>,
```

Comment:
Please add a comment here that Isolate must outlive Inspector

Reply:
I think that's not a problem any more since it's stored in runtime state which can't outlive the isolate. The reason it's coincidentally an `Option` is so it can be dropped when snapshotting, same as `global_realm: Option<JsRealm>`.

Reply:
Okay, that makes sense. Can you address the exception queue in this PR?